src/calculateRentalCost.js

```diff
@@ -28,7 +28,21 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const costForOneDay = 40;
+  const discountForThreeDays = 20;
+  const discountForSevenDays = 50;
+
+  if (days < 3) {
+    return costForOneDay * days;
+  }
+
+  if (days >= 3 && days < 7) {
+    return costForOneDay * days - discountForThreeDays;
+  }
+
+  if (days >= 7) {
+    return costForOneDay * days - discountForSevenDays;
+  }
 }
```

Comment:
Can save two lines here - it will work anyway after first two checks.
```suggestion
  return costForOneDay * days - discountForSevenDays;
}
```